web/app/themes/clarity/inc/mymoj.php

```diff
@@ -249,69 +249,7 @@ public static function get_apps($agency = 'hq')
             'external' => true,
           ],
         ],
-        'judicial-appointments-commission' => [
```

Comment:
Should this be changed to JAC rather then removed?

Reply:
I opted to just create a new entry after the OSPT using the JAC prefix instead and then after go through and removed all references to this old one. I guess I could have replaced it just as well. I've found though with the intranet it is sometimes better to create a new entry and then when it is working go through and remove the old one.